mutt/buffer.c

```diff
@@ -264,14 +264,14 @@ bool mutt_buffer_is_empty(const struct Buffer *buf)
  */
 void mutt_buffer_alloc(struct Buffer *buf, size_t new_size)
 {
-  if (!buf)
+  if (!buf) 
```

Comment:
Careful!  Some trailing whitespace has crept in.

Running `clang-format` on your changed code will fix this
(and check other things like indentation and `{}`-placement.